src/GatewayClient.cpp

```diff
@@ -90,7 +90,8 @@ bool GatewayClient::loop() {
 void GatewayClient::_sendKeepAlive() {
   ESP_LOGV(TAG, "Sending keep alive message");
 
-  OpenShock::Serialization::KeepAlive keepAlive(OpenShock::Millis());
+  // Casting to uint64 here is safe since millis is guaranteed to return a positive value
+  OpenShock::Serialization::KeepAlive keepAlive((std::uint64_t)OpenShock::millis());
```

Comment:
Why is keepalive uint64?

Reply:
Because there is no reason it should be int64, and the current LCG API uses this now.

Additionally flatbuffers wont allow empty table/struct.
So having a timer could be useful in future.

Maybe to see board uptime or something